2021/CVE-2021-25955.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
Expected location (2021/25xxx/CVE-2021-25955.json) of file does not match actual location (2021/CVE-2021-25955.json)

Reply:
Changed to 2021/25xxx